app/views/fae/application/_content_form.html.slim

```diff
@@ -1,5 +1,6 @@
 ruby:
   require_locals     ['attribute', 'f'], local_assigns
+  blank_label          = !label || label == '' unless label.nil?
```

Comment:
I find this line difficult to interpret. If we focus on supporting `label: false`, then it could look something like:

```ruby
has_label = !local_assigns[:label].eql?(false)
```

This is a pattern we've using in other partials too.

Reply:
Nice that looks better. `label: false` was the only way to remove the label for simpleform hence the check for false or an empty string

Reply:
@jamesmk what do you about using `![false, ''].include?(local_assigns[:label])` instead?

The problem with your way is that if `label: false`, then `!local_assigns[:label]` will return true when really it should be false.